Can not import @uploadcare/react-widget/en-min/

### Describe the bug

Neither via NPM nor via YARN can I import / add `@uploadcare/react-widget/en-min/` as described in the README.

### Expected behavior
The installation should work.

### Code / screenshots

`npm install @uploadcare/react-widget/en-min/` -> returns 
```
npm ERR! code ENOENT
npm ERR! syscall open
```

`yarn add @uploadcare/react-widget/en-min/` -> returns:
```
error An unexpected error occurred: "https://registry.yarnpkg.com/@uploadcare%2freact-widget/en-min/: 
Request \"https://registry.yarnpkg.com/@uploadcare%2freact-widget/en-min/\" returned a 405".
```

### Environment

 - Library version: 
 - Language/framework version: NPM 8.5.0, YARN 1.22.19
 - OS version:  macOS 12.3.1

--- a/README.md
+++ b/README.md
@@ -68,9 +68,9 @@ You can refer to our [integration guide][react-guide] for more details.
 By default, npm and other package managers import the full (all locales) CommonJS or ESM bundle.
 
 To reduce your bundle size, you can also import one of the following:
-* The english-only bundle (saves ~27% in bundle size) as `@uploadcare/react-widget/en/index.js`
-* The minified all-locales bundle (saves ~44% in bundle size) as `@uploadcare/react-widget/min/index.js`
-* The minified english-only bundle (saves ~60% in bundle size) as `@uploadcare/react-widget/en-min/index.js`
+* The english-only bundle (saves ~27% in bundle size) as `@uploadcare/react-widget/en`
+* The minified all-locales bundle (saves ~44% in bundle size) as `@uploadcare/react-widget/min`
+* The minified english-only bundle (saves ~60% in bundle size) as `@uploadcare/react-widget/en-min`
 
 ## Configuration
 

--- a/package.json
+++ b/package.json
@@ -3,10 +3,19 @@
   "version": "2.1.3",
   "description": "React component for Uploadcare Widget",
   "type": "module",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "types/index.d.ts",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./types/index.d.ts",
   "sideEffects": false,
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./en": "./en/index.js",
+    "./en-min": "./en-min/index.js",
+    "./min": "./min/index.js"
+  },
   "scripts": {
     "lint:types": "dtslint types --onlyTestTsNext",
     "lint:js": "eslint ./src/** ./test/** rollup.config.js",

